extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js

```diff
@@ -541,21 +551,98 @@ export class AmpLightboxViewer extends AMP.BaseElement {
     const tagName = this.elementsMetadata_[this.currentElemId_]
         .tagName;
     if (tagName === 'AMP-IMG') {
-      this.resizeImageViewerDimensions_();
+      this.resizeImageViewerDimensions_().then(() => this.enter_(element));
     }
     this.updateDescriptionBox_();
   }
 
+  /**
+   * Entry animation to transition in a lightboxable image
+   * @param {!Element} sourceImage
+   */
+  // TODO (cathyxz): make this generalizable to more than just images
+  enter_(sourceImage) {
+    st.setStyles(this.element, {
+      opacity: 0,
+      display: '',
+    });
+
+    const anim = new Animation(this.element);
+    const dur = 500;
```

Comment:
make const

Reply:
Sorry, make what const?

Reply:
Sorry, meant: move to a top level constant